dhcpv6/ztpv6/parse_vendor_options.go

```diff
@@ -92,3 +100,32 @@ func ParseVendorData(packet dhcpv6.DHCPv6) (*VendorData, error) {
 	}
 	return nil, errors.New("failed to parse vendor option data")
 }
+
+type MlnxSubOption int
+
+const (
+	MlnxSubOptionModel   MlnxSubOption = 1
+	MlnxSubOptionPartNum MlnxSubOption = 2
+	MlnxSubOptionSerial  MlnxSubOption = 3
+	MlnxSubOptionMac     MlnxSubOption = 4
+	MlnxSubOptionProfile MlnxSubOption = 5
+	MlnxSubOptionRelease MlnxSubOption = 6
+)
+
+func parseMellanoxVendorData(vendorOptsOption *dhcpv6.OptVendorOpts) (*VendorData, error) {
```

Comment:
This name is tricky. I thought this was implementing an alternative vendor option parser to the one defined in here:

https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/option_vendor_opts.go#L56

Maybe `(get)MellanoxVendorData`?